com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -499,11 +500,15 @@ public int compare(String i1, String i2) {
         return sortedInputs;
     }
 
-    private List<String> loadDefoldIgnore() throws CompileExceptionError {
+    private void loadIgnoredFilesAndFolders() throws CompileExceptionError {
+        String excludeFoldersStr = this.option("exclude-build-folder", "");
```

Comment:
I'm not at all certain of this one.
We need to keep it forever due to some external plugin not working the way they want?

Reply:
I don't know why we have it. 
Here is PR https://github.com/defold/defold/pull/5497
If you remember why it was added let me know. If you think we don't need it anymore- I can remove it.

Reply:
Now it's marked as deprecated